.readthedocs.yml

```diff
@@ -12,7 +12,7 @@ build:
         post_install:
             # VIRTUAL_ENV needs to be set manually for now.
             # See https://github.com/readthedocs/readthedocs.org/pull/11152/
-            - VIRTUAL_ENV=$READTHEDOCS_VIRTUALENV_PATH uv pip install .[docs]
+            - VIRTUAL_ENV=$READTHEDOCS_VIRTUALENV_PATH pip install .[docs]
```

Comment:
Readthedocs fails with

```
...
running build_rust
--- stderr:
error: [Errno 2] No such file or directory: 'cargo'
---
```

for some reason. Is this ok as temporary making this build for now & optimize again later by fixing the use of uv?